### PR TITLE
[#336] Added wrapping of fallback closure with response entity

### PR DESCRIPTION
**Pull request by marcingrzejszczak**

### CODE REVIEW

File: micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/resttemplate/fluent/common/response/executor/RestExecutor.groovy [hunk 1]
```diff
@@ -118,7 +118,7 @@ final class RestExecutor<T> {
 
                     @Override
                     protected ResponseEntity<T> getFallback() {
-                        return hystrixFallback.call()
+                        return wrapWithResponseEntity(hystrixFallback.call())
```
Comment (szpak) [inline micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/resttemplate/fluent/common/response/executor/RestExecutor.groovy]: Maybe it is worth to add comment next to `@TypeChecked` to not replace it with `@CompileStatic` due to dynamic dispatch?

File: micro-infra-spring-base/src/test/groovy/com/ofg/infrastructure/web/resttemplate/fluent/ServiceRestClientSpec.groovy [hunk 2]
```diff
@@ -22,11 +16,7 @@ import spock.lang.Specification
 import java.util.concurrent.Executors
 import java.util.concurrent.ScheduledExecutorService
 
-import static org.springframework.http.HttpMethod.DELETE
-import static org.springframework.http.HttpMethod.GET
-import static org.springframework.http.HttpMethod.HEAD
-import static org.springframework.http.HttpMethod.POST
-import static org.springframework.http.HttpMethod.PUT
+import static org.springframework.http.HttpMethod.*
```
Comment (szpak) [inline micro-infra-spring-base/src/test/groovy/com/ofg/infrastructure/web/resttemplate/fluent/ServiceRestClientSpec.groovy]: What is a standard approach for imports? `*`?

**Comment by szpak** [comment]
Besides comments, +1

File: micro-infra-spring-base/src/test/groovy/com/ofg/infrastructure/web/resttemplate/fluent/ServiceRestClientSpec.groovy (hunk 2 above)
Comment (marcingrzejszczak) [inline micro-infra-spring-base/src/test/groovy/com/ofg/infrastructure/web/resttemplate/fluent/ServiceRestClientSpec.groovy]: Yours is ok - I must fix my home IDEA settings

**Pull request merged**